### DISCUSSION TITLE
RustDesk Pro behind Fortigate

### DISCUSSION THREAD

**Discussion by DonoJulien** [rustdesk/rustdesk-server-pro#460] [General]
Hello,

We recently installed a RustDesk Server Pro on a Debian VM, on an ESXI server.
We normally have no problems at all with NAT or anything like that, as we directly manage them on our Fortigate.

I took the time to do a drawing of the situation, no worries any IP are either redacted or fake.

I want to access customers computers, so they have to directly point to the RustDesk server.
When we use local IP in the LAN, everything works fine, but as soon as we replace the local IP with the DNS entry, nothing works (but logging with accounts works ...)

![image](https://github.com/user-attachments/assets/a868fd99-3288-443a-b212-a80ba4fe36e2)

If anyone have a clue, I'll take it !

**Comment by rustdesk** [comment]
This might help, https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet

**Reply by rustdesk** [reply]
and https://discord.com/channels/804630702657110016/804630702657110018/1210636199782522991

**Reply by DonoJulien** [reply]
Thanks for your fast answer, I basically am dumb and feel sorry for bothering you.
Everything was setup correctly, I just had another NAT rule overlapping the same ports but redirecting to a completely random server.

Thanks again
